src/linker/Linker.Dataflow/ValueNode.cs

```diff
@@ -332,20 +113,6 @@ public void Dispose ()
 	/// </summary>
 	public abstract class LeafValueNode : ValueNode
```

Comment:
Is the type needed?

Reply:
Not right now, but I intentionally left the type design same as before to avoid too much churn. I will have a separate PR to rework the value type system.